lib/fog/aws/requests/storage/head_object_url.rb

```diff
@@ -0,0 +1,40 @@
+module Fog
+  module Storage
+    class AWS
+      module HeadObjectUrl
+        def head_object_url(bucket_name, object_name, expires, options = {})
```

Comment:
Wondering wither this and duplicate code in get_object_url should be put in module?